scripts/build_containers.sh

```diff
@@ -161,10 +161,10 @@ function prune_containers {
 
 trap prune_containers ERR INT EXIT
 
-echo "Branch: $BUILDKITE_BRANCH"
-echo "PR    : $BUILDKITE_PULL_REQUEST"
-echo "Commit: $BUILDKITE_COMMIT"
-echo "Tag   : $BUILDKITE_TAG"
+if [ -v BUILDKITE_BRANCH ]; then echo "Branch: $BUILDKITE_BRANCH"; fi
```

Comment:
According to Buildkite's documentation, there will be a single environment variable called `BUILDKITE` which is always true. Just in case it's sufficient to wrap the whole block in a single `if` instead of every variable individually.